GViewCore/src/App/Instance.cpp

```diff
@@ -97,9 +97,9 @@ bool Instance::Init()
 
     return true;
 }
-bool Instance::Add(std::unique_ptr<AppCUI::OS::IFile> file, const AppCUI::Utils::ConstString& name, std::string_view ext)
+bool Instance::Add(std::unique_ptr<AppCUI::OS::IFile> file, const AppCUI::Utils::ConstString& path, std::string_view ext)
```

Comment:
Nu prea e ok (semantic) cu path in loc de name
Functia asta primeste deja un IFile si e generica (acel IFile poate fi peste orice -> path local / memorie / etc) ==> de asta am si spus ca e name si nu path

Reply:
fixed